reservation_units/utils/reservation_unit_pricing_helper.py

```diff
@@ -94,6 +95,41 @@ def check_pricing_counts(cls, is_draft: bool, data: Dict[str, Any]):
                 "only ACTIVE and FUTURE pricings can be mutated"
             )
 
+    @classmethod
+    def calculate_vat_prices(cls, data: Dict[str, Any]) -> Dict[str, Any]:
+        """Calculates vat prices from net prices and returns a dict of pricing data."""
+        pricings = data.get("pricings", [])
+
+        for pricing in pricings:
+            highest_price_net = pricing.get("highest_price_net", 0)
+            lowest_price_net = pricing.get("lowest_price_net", 0)
+
+            if highest_price_net < lowest_price_net:
+                raise serializers.ValidationError(
+                    "Highest price cannot less than lowest price."
+                )
+
+            tax_percentage = pricing.get("tax_percentage")
```

Comment:
```suggestion
            tax_percentage = pricing.get("tax_percentage", 0)
```

That way you could drop the falsy check from the next `if` statement.

Reply:
Just realized this comment is complete nonsense. Ignore 😄